### ISSUE TITLE
Front-end editing not for fields on non-publish tabs

### ISSUE THREAD

**Issue by benoitlemay** [ExpressionEngine/ExpressionEngine#4544]
Frontend edit- if the field is in a channel layout tab? When you frontend edit, the pane is empty.  (aka I can't front-edit Grid Fields.. )
EE 7.5.1
PHP 7.4 (tried 8.3 also)

(Note- summary edited by Robin)

To replicate, grab a channel, make a layout, move a field or two over to a different tab- aka not the publish tab.  That's it, now the field won't be editable on the frontend.  Tried moving to the existing category tab, made a new tab, tried moving it to that.  Either way, no edit.

Now- go frontend edit.  When you click the pencil for the field in the tab, the edit modal shows a la:

<img width="1193" alt="Screenshot 2024-10-25 at 1 13 13 PM" src="https://github.com/user-attachments/assets/b6eace2c-ea5a-4438-8ae4-9d79cd0836c0">

There's no js error when you do this (no php errors either).  BUT interestingly, if I go to edit another field on the page- one not in a tab- it opens fine, but now I get js errors!

<img width="1190" alt="Screenshot 2024-10-25 at 1 13 49 PM" src="https://github.com/user-attachments/assets/20058aa7-def0-4a21-be6a-ffbd28bd3d8a">

--- original report below ---

Front-edit is enable for the field..

In my rendered page i can edit the title and many other fields... I see the pencil icon to edit the grid but modal is empty
![image](https://github.com/user-attachments/assets/f84605e3-0cfc-42c3-b679-58d7ecd567e4)

In a test page.. (see image) i can edit the title.. but not the grid..  JUST TO CLARIFY.. THE DATE AND CODE (AFTER THE DATE) ARE IN THE GRID
![image](https://github.com/user-attachments/assets/3010d125-3494-4dfe-a4af-7df92bd0fab5)

There's no error in the Web Console.

**Comment by robinsowell** [comment]
Hrm, I just did a test on 7.5 and a grid comes up for me.

<img width="964" alt="Screenshot 2024-10-22 at 12 29 33 PM" src="https://github.com/user-attachments/assets/4fb74ba6-eae5-4a4c-8aff-4dc7581d7846">

Are you getting any javascript errors? Do you have any other grid fields you can test with? I'm wondering if there's something specific about that grid.

**Comment by benoitlemay** [comment]
I have many grid field in the same channel... all are NOT working..  File Grid in an other channel is working great.. (need to move the file grid tag outside the <audio> tag but it works..  TEsting few things and a date field (inside the grid) show error.. 
![image](https://github.com/user-attachments/assets/4749e27c-6b39-40bb-b50d-ba5623381b90). 

Tested with grid with only Input textarea, RTE and toggle and not working..Pencil is showing up but modal is empty NO error in the console

**Comment by robinsowell** [comment]
Hrm , ok, let's see if we can get a php error.

Can you do this- ftp in, go to system/user/ and create a writable folder called logs - then go to the config/config.php file and add this:

```
$config['log_date_format'] = 'Y-m-d H:i:s';
$config['log_threshold'] = '1';
```

So it should log errors to a file it creates in that new logs folder.

Go fire up frontend editing and open your grid field- anything?

Also- what add-ons are you using? Transcribe or Publisher? And big ones?

And it's weird- because the file grid shares a ton of code with the grid- surprised to see it working and grid not!

**Comment by benoitlemay** [comment]
Nothing is written in the log folder.. 
EE Addons are : Channel Video, Pro Var, Request, RTE..

More TESTS... 
In Firefox (and Chrome) i got this error `Uncaught TypeError: t.$refs[e] is undefined
    init https://dev.levoyagistedequebec.com/themes/ee/pro/js/fronteditor.min.js?v=2.1.0-1729621294:1
` as soon as i click on a pencil .. BUT NOT THE FIRST TIME I LOAD THE PAGE..

**Comment by robinsowell** [comment]
REALLY WEIRD!  Did you get a chance to turn on the php error logging?  And if we don't get anything from that, you can hit me in support@expressionengine.com with superadmin login info and a link to a good example page and I'll login for a closer look.

But definite- see if the error logging gets us anything.

**Comment by benoitlemay** [comment]
Email sent

**Comment by robinsowell** [comment]
AHA- so I don't have it solved, but I can see what makes it happen.

Anything in a channel layout tab? Doesn't show in the edit pane on frontend editing.

So if I took admins off of your layout? Everything was editable.

If I made a duplicate of your field with no layout- everything editable.  If I made a tab and moved the prix grid into the new tab? Blank pane inside the edit modal.  SAME if I did it with a textarea field.  So I don't even think it's a grid thing, I think it's purely a layout tab thing.

I'm going to tweak the bug report to make it more clear.
